いくつかのファイルでLintエラー

## 概要

Linterで`playwright.config.ts`と`validation.spec.ts`でエラー発生

## 再現手順

1. npm run lint

## 期待する動作

エラーなし

## 実際の動作

```shell
jpmap_terrain/playwright.config.ts
  0:0  error  Parsing error: "parserOptions.project" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided project(s): playwright.config.ts

jpmap_terrain/tests/validation.spec.ts
  0:0  error  Parsing error: "parserOptions.project" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided project(s): tests/validation.spec.ts
```

## 作業完了の定義

Lintのエラーなし

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -17,6 +17,10 @@
         "skipLibCheck": true,         // node_modules内の型エラーで止まらないために必須
         "isolatedModules": true       // 他のトランスパイラとの互換性を確保
     },
-    "include": ["src/**/*"],
+    "include": [
+        "src/**/*",
+        "playwright.config.ts",
+        "tests/**/*.ts"
+    ],
     "exclude": ["node_modules/**", "dist/**", "webpack.*"]
 }

